app/services/admin/FeatureSwitchService.scala

```diff
@@ -65,15 +65,26 @@ class FeatureSwitchService @Inject()(
     }
   }
 
-  def set(featureSwitchName: FeatureSwitchName, enabled: Boolean): Future[Boolean] =
+  def set(featureSwitchName: FeatureSwitchName, enabled: Boolean): Future[Boolean] = {
+    Logger("application").info(s"Setting feature switch ${featureSwitchName.name} to ${enabled.toString}")
     if (appConfig.readFeatureSwitchesFromMongo) {
       featureSwitchRepository.setFeatureSwitch(featureSwitchName, enabled)
     } else {
       setFS(featureSwitchName, enabled)
       Future.successful(true)
     }
+  }
 
-  def setAll(featureSwitches: Map[FeatureSwitchName, Boolean]): Future[Unit] =
-    featureSwitchRepository.setFeatureSwitches(featureSwitches)
+  def setAll(featureSwitches: Map[FeatureSwitchName, Boolean]): Future[Unit] = {
+    Logger("application").info(s"Setting all feature switches. FS values: $featureSwitches")
+    if (appConfig.readFeatureSwitchesFromMongo) {
+      featureSwitchRepository.setFeatureSwitches(featureSwitches)
+    } else {
+      featureSwitches.foreach(featureSwitch =>
+        setFS(featureSwitch._1, featureSwitch._2)
```

Comment:
resolving tuples in this fashion is generally discouraged as its hard to read/understand these lines